src/store/actions.js

```diff
@@ -47,6 +50,9 @@ import {
 } from './controllerSubscriptions'
 import initialState from './state'
 
+const ec = elliptic.ec('secp256k1')
+
+window.torus = torus
```

Comment:
Pls no.
Use torus.ec instead of creating elliptic instance

Reply:
KEKL